pybossa/cache/projects.py

```diff
@@ -193,6 +195,14 @@ def search_lock_status_sorting_result():
         params["assign_user"] = args["sql_params"]["assign_user"]
 
         order_by =  args.get('order_by') or "priority_0 DESC"
+
+        if 'in_progress' in order_by:
+            if args.get('order_by') == 'in_progress asc' or args.get('order_by') == 'in_progress desc':
+                order_by = "priority_0 DESC"
+            else:  # multiple conditions - need remove clause "in_progress asc"
+                order_by = re.sub(r"(in_progress (desc|asc)\,?\s*)", "", order_by)
+                order_by = order_by.rstrip().rstrip(',')
```

Comment:
This code gives an impression that the sorting for in_progress items are happening on priority. As discussed, there's other place wherein the score applied by ttl to each in_progress items that guarantees that sorting correctly happens based on in_progress. Please include comments explaining this so that the logic is recorded for future references. Thanks!